.eslintrc.json

```diff
@@ -58,7 +58,10 @@
         "@typescript-eslint/comma-dangle": ["error", "only-multiline"],
         "@typescript-eslint/explicit-module-boundary-types": "off",
         "@typescript-eslint/ban-ts-comment": "off",
-        "@typescript-eslint/ban-types": "off"
+        "@typescript-eslint/ban-types": "off",
+        "react/require-default-props": "off",
+        "react/no-unused-prop-types": "off",
```

Comment:
eslint is only unhappy about`no-unused-prop-types` in `components/Navigation/NavItem/NavItem.tsx` component.

It's better to disable it just for that component (until it's fixed).

You can make it my wrapping the `type Props = {}` block

```ts
/* eslint-disable react/no-unused-prop-types */
export interface Props extends React.HTMLAttributes<HTMLDivElement> {
    ...
}
/* eslint-enable react/no-unused-prop-types */

```

Reply:
:+1: